compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -1237,6 +1237,8 @@ pub trait PrettyPrinter<'tcx>:
                 self.pretty_print_bound_var(debruijn, bound_var)?
             }
             ty::ConstKind::Placeholder(placeholder) => p!(write("Placeholder({:?})", placeholder)),
+            // FIXME(julianknodt): write out some legible representation of an abstract const?
+            ty::ConstKind::Abstract(_ac) => p!("[Const Expr]"),
```

Comment:
leave a fixme there for now, i remember @b-naber working on something like this for `AbstractConst`s before, though that didn't get merged.